File: Q9.py
idade = list(range(20))
vezes = 1
soma = 0 
for i in range(0, len(idade)):
    idade[i] = int(input(f"Digite a {vezes}ª idade: "))
    soma = soma + idade[i]
    vezes += 1
média = int(soma / 20)
print(f"A média das 20 idades que você digitou é {média} anos.")

File: Q9.java
import java.util.Scanner;

public class Q9
{
    public static Scanner leia = new Scanner(System.in);
    public static void main(String args[])
    {
        int idade[] = new int[20], soma = 0, vezes = 1, média;
        for (int i = 0; i < idade.length; i ++, vezes ++)
        {
            System.out.printf("Digite a %dª idade: ", vezes);
            idade[i] = leia.nextInt();
            soma = soma + idade[i];
        }
        média = soma / 20;
        System.out.printf("A média das 20 idades que você digitou é %d anos.\n", média);
    }
}